web/profiles/sdd/sdd.info.yml

```diff
@@ -1,21 +1,21 @@
 name: SDD
 type: profile
 description: 'Skill drupal development installation profile for content projects'
-core: 8.x
+core_version_requirement: ^9
 
 dependencies:
   - automated_cron
   - big_pipe
   - block
   - block_content
-  - block_content_permissions
+#  - block_content_permissions
```

Comment:
Please don't comment here, remove the lines and we can create issues to migrate those modules to d9

Reply:
We need to fix any of this modules, or add patches but the list should not be affected